user/enterprise/ssl-certificate-management.md

```diff
@@ -88,131 +88,167 @@ Upon restart you can then verify whether SSL verification is now working. If you
+  ```bash
+  sudo add-apt-repository ppa:certbot/certbot
+  ```
+3. Update available packages:
+  ```bash
+  sudo apt-get update
+  ```
+3. Install required certbot dependency:
+  ```bash
+  sudo apt-get install software-properties-common
+  ```
+5. Install the certbot package:
+  ```bash
+  sudo apt-get install certbot
+  ```
+
+### Generating a new Let's Encrypt certificate
+
+The certbot tool offers multiple ways to obtain a certificate. We'll pick the 'temporary webserver' option since it doesn't require any additional setup or configuration. The only prerequisite is that the Travis CI Enterprise container must be stopped so that the certbot webserver can bind properly to port 443.
+
+> Your Travis CI Enterprise instance will be unavailable during this process.
+
+To generate your certificate:
+
+1. Open a ssh connection to the platform machine.
+2. Stop your Travis CI Enterprise:
+  ```bash
+  replicatedctl app stop
+  ```
+3. Start the interactive certificate process:
+  ```bash
+  sudo certbot certonly
+  ```
+4. At the prompt select option `1` to `Spin up a temporary webserver`:
+  ```bash
+  How would you like to authenticate with the ACME CA?
+  -------------------------------------------------------------------------------
+  1: Spin up a temporary webserver (standalone)
+  2: Place files in webroot directory (webroot)
+  -------------------------------------------------------------------------------
+  Select the appropriate number [1-2] then [enter] (press 'c' to cancel): 1
+  ```
+5. Fill in your email address:
+  ```bash
+  Enter email address (used for urgent renewal and security notices) (Enter 'c' to
+  cancel): ops@example.com
+  ```
+6. Accept the Terms of Services:
+  ```bash
+  -------------------------------------------------------------------------------
+  Please read the Terms of Service at
+  https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf. You must agree
+  in order to register with the ACME server at
+  https://acme-v01.api.letsencrypt.org/directory
+  -------------------------------------------------------------------------------
+  (A)gree/(C)ancel: A
+  ```
+7. Decide if you'd like to share your email address with the EFF (this decision has no effect on your certificate or your Travis CI Enterprise instance):
+  ```bash
+  -------------------------------------------------------------------------------
+  Would you be willing to share your email address with the Electronic Frontier
+  Foundation, a founding partner of the Let's Encrypt project and the non-profit
+  organization that develops Certbot? We'd like to send you email about EFF and
+  our work to encrypt the web, protect its users and defend digital rights.
+  -------------------------------------------------------------------------------
+  (Y)es/(N)o: N
+  ```
+8. Provide your domain name:
+  ```bash
+  Please enter in your domain name(s) (comma and/or space separated)  (Enter 'c'
+  to cancel): travis.example.com
```

Comment:
should probably use the same example domain name throughout?

Reply:
IF you could be bothered that might apply the entire Enterprise docs, but not worries :-D

Reply:
I think this is a good idea and I don't think it would be too much work overall. What exactly were you thinking in terms of the value? In other parts of this page specifically we use this: `https://<your-travis-ci-enterprise-domain>:8800`. I did this since that is what we used in the recently updated setup steps.

In this specific instance I was going to change `travis.example.com` to `<your-travis-ci-enterprise-domain>` before hesitating and wondering if that was the best call. Is this confusing to you?

```sh
Please enter in your domain name(s) (comma and/or space separated)  (Enter 'c'
to cancel): <your-travis-ci-enterprise-domain>
```

One thing I have struggled with in the enterprise docs as a customer is not understanding when I am expected to include the protocol and when I am not. It this specific example we want `travis.example.com` and not `https://travis.example.com`. 

Does using `<your-travis-ci-enterprise-domain>` when we don't want a protocol make that clear?

Reply:
Yeah, it is an old chestnut. 

travis.example.com vs travis.<yourdomain>.com vs <your-travis-ci-enterprise-domain>

I have a preference for the last, because it might look different to the first two. 

The protocol is a different thing again, and super valid. Where possibly I'd advocate refactoring the product about always asking for it, whether or not you use it or something :-p

Reply:
I agree with that last point, definitely. Removing the question entirely is the best option, in my opinion. 😄 

In the meantime I will try updating with `<your-travis-ci-enterprise-domain>` and we will see what we think.

Reply:
Alright, done in 0ed4650b. I think it looks pretty good overall. What do you think?

Reply:
Looks great, I'll give it another once over tomorrow, and merge in :100:

Reply:
Thank you!